app/scripts/views/sign_up.js

```diff
@@ -225,6 +227,17 @@ function (Cocktail, _, p, BaseView, FormView, Template, AuthErrors, mailcheck,
       }
       return this._isMailcheckEnabledValue;
     },
+    _isMailcheckUseful: function () {
```

Comment:
naming nit: we tend to use "isFoo" for state-checking things like "isMailcheckEnabled" and "isSignUpDisabled"; since this is an after-the-fact check on what happened, perhaps it should be named something like "was" rather than "is".